src/app/shared/components/template/components/select-text/select-text.component.html

```diff
@@ -1,4 +1,7 @@
 <div class="box-wrapper" [attr.data-toggled]="toggle" (click)="triggerCopyContent($event, _row)">
+  <div *ngIf="QRCodeValue">
```

Comment:
I don't think I quite understand what is going on with the select-text component and why it receives QR codes as an input... Is there a reason why we're not just using the qrcode template component either before/after the select-text component?

Reply:
I think ask @EsmeeIDEMS if happy with the QR code sitting outside.

If yes then let's remove the QR-specific code from the select to keep it cleaner and merge

If no I'd suggest creating a display group with the QR and select-text components inside, removing the styles from select-text and instead applying at the display group level (so the display group adds a border around everything), so that again we do not need any qr-specific code in the select-text component

Reply:
OK, I'm happy with the QR code outside the box too, so I've updated the code and PR description, I think it's ready to merge.

I think that what you're describing, having "copy to clipboard" available as a template action, would be achievable. But I think it would be non-trivial, for the reasons you suggest as well as handling the custom styling that gets applied to the whole dashed box currently. Perhaps it's worth creating a low priority issue?